src/traces/sankey/render.js

```diff
@@ -970,88 +961,55 @@ module.exports = function(gd, svg, calcData, layout, callbacks) {
         .ease(c.ease).duration(c.duration)
         .call(sizeNode);
 
-    var nodeCapture = sankeyNode.selectAll('.' + c.cn.nodeCapture)
-        .data(repeat);
-
-    nodeCapture.enter()
-        .append('rect')
-        .classed(c.cn.nodeCapture, true)
-        .style('fill-opacity', 0);
-
-    nodeCapture
-        .attr('x', function(d) {return d.zoneX;})
-        .attr('y', function(d) {return d.zoneY;})
-        .attr('width', function(d) {return d.zoneWidth;})
-        .attr('height', function(d) {return d.zoneHeight;});
-
-    var nodeCentered = sankeyNode.selectAll('.' + c.cn.nodeCentered)
-        .data(repeat);
-
-    nodeCentered.enter()
-        .append('g')
-        .classed(c.cn.nodeCentered, true)
-        .attr('transform', nodeCentering);
-
-    nodeCentered
-        .transition()
-        .ease(c.ease).duration(c.duration)
-        .attr('transform', nodeCentering);
-
-    var nodeLabelGuide = nodeCentered.selectAll('.' + c.cn.nodeLabelGuide)
-        .data(repeat);
-
-    nodeLabelGuide.enter()
-        .append('path')
-        .classed(c.cn.nodeLabelGuide, true)
-        .attr('id', function(d) {return d.uniqueNodeLabelPathId;})
-        .attr('d', textGuidePath)
-        .attr('transform', sankeyInverseTransform);
-
-    nodeLabelGuide
-        .transition()
-        .ease(c.ease).duration(c.duration)
-        .attr('d', textGuidePath)
-        .attr('transform', sankeyInverseTransform);
-
-    var nodeLabel = nodeCentered.selectAll('.' + c.cn.nodeLabel)
+    var nodeLabel = sankeyNode.selectAll('.' + c.cn.nodeLabel)
         .data(repeat);
 
     nodeLabel.enter()
         .append('text')
         .classed(c.cn.nodeLabel, true)
-        .attr('transform', textFlip)
-        .style('cursor', 'default')
-        .style('fill', 'black');
+        .style('cursor', 'default');
 
     nodeLabel
-        .style('text-shadow', function(d) {
-            return d.horizontal ? svgTextUtils.makeTextShadow(1, '#fff') : 'none';
+        .attr('data-notex', 1) // prohibit tex interpretation until we can handle tex and regular text together
```

Comment:
Nonblocking, but this isn't quite the rationale here, here we don't want tex because of shadows and positioning issues :)